tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -623,3 +625,57 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+def validate__WORK_ELIGIBLE_INDICATOR__HOH__AGE():
+    """If WORK_ELIGIBLE_INDICATOR == 11 and AGE < 19, then RELATIONSHIP_HOH != 1."""
+    # value is instance
+    def validate(instance):
+        false_case = (False,
+                      "If WORK_ELIGIBLE_INDICATOR == 11 and AGE < 19, then RELATIONSHIP_HOH != 1",
+                      ['WORK_ELIGIBLE_INDICATOR', 'RELATIONSHIP_HOH', 'DATE_OF_BIRTH']
+                      )
+        true_case = (True,
+                     None,
+                     ['WORK_ELIGIBLE_INDICATOR', 'RELATIONSHIP_HOH', 'DATE_OF_BIRTH'],
+                     )
+        try:
+            WORK_ELIGIBLE_INDICATOR = (
+                instance["WORK_ELIGIBLE_INDICATOR"]
+                if type(instance) is dict
+                else getattr(instance, "WORK_ELIGIBLE_INDICATOR")
+            )
+            RELATIONSHIP_HOH = (
+                instance["RELATIONSHIP_HOH"]
+                if type(instance) is dict
+                else getattr(instance, "RELATIONSHIP_HOH")
+            )
+            RELATIONSHIP_HOH = int(RELATIONSHIP_HOH)
+
+            DOB = (
+                instance["DATE_OF_BIRTH"]
+                if type(instance) is dict
+                else getattr(instance, "DATE_OF_BIRTH")
+            )
+
+            DOB = str(DOB)
+            DOB_datetime = datetime.datetime.strptime(DOB, '%Y%m%d')
+            today = datetime.date.today()
+            AGE = today.year - DOB_datetime.year - ((today.month, today.day) < (DOB_datetime.month, DOB_datetime.day))
```

Comment:
@elipe17 Age calculation is relative to the reporting period (`RPT_MONTH_YEAR`). per the workbook:

`requires calculating AGE: DOB (item #32) must be valid; convert RPT_MON_YEAR (item #4) to YYYYMMDD and set DD to 01; AGE formula: (rpt_month_year - DOB)/365.25`